BUILDING.md

```diff
@@ -253,6 +253,23 @@ To install this version of Node.js into a system directory:
 $ [sudo] make install
 ```
 
+#### Building a debug build
+
+If you run into a issue where Node.js segfaults and the "segfault-handler" does not provide enough information, you might have to build a debug build of Node.js to get enough information to debug the issue further, this is done by :
```

Comment:
Please replace 'a issue' -> 'an issue'.

Reply:
Can I suggest:
```
If you run into a issue where Node.js does not provide enough debug information, you can try to build a debug enabled binary:
```

Reply:
@refack I’d prefer something that makes clear that this relates almost exclusively to native errors.

Reply:
```
If you run into a issue where the information provided by the JS stack trace is not enough, or if you suspect the error happens outside of the JS VM, you can try to build a debug enabled binary.
```